docs/src/main/sphinx/connector/duckdb.md

```diff
@@ -11,19 +11,37 @@ DuckDB instances.
 
 ## Configuration
 
-To configure the DuckDB connector, create a catalog properties file
-in `etc/catalog` named, for example, `example.properties`, to
-mount the DuckDB connector as the `duckdb` catalog.
-Create the file with the following contents, replacing the
-connection properties as appropriate for your setup:
+The connector can query a DuckDB database. Create a catalog properties file that
+specifies the DuckDb connector by setting the `connector.name` to `duckdb`.
+
+For example, to access a database as the `example` catalog, create the file
+`etc/catalog/example.properties`. Replace the connection properties as
+appropriate for your setup:
 
 ```none
 connector.name=duckdb
-connection-url=jdbc:duckdb://<path>
-connection-user=root
-connection-password=secret
+connection-url=jdbc:duckdb:<path>
 ```
 
+The `connection-url` defines the connection information and parameters to pass
+to the DuckDB JDBC driver. The parameters for the URL are available in the
+[DuckDB JDBC driver documentation](https://duckdb.org/docs/clients/java.html).
+
+The `<path>` must point to an existing, persistent DuckDB database. For example,
+use `jdbc:duckdb:/opt/duckdb/trino.duckdb` for a database created with the
+command `duckdb /opt/duckdb/trino.duckdb`. The database automatically contains
+the `main` schema  and the `information_schema` schema. Use the `main` schema
+for your new tables or create a new schema.
+
+When using the connector on a Trino cluster the path must be consistent on all
+nodes and point to a shared storage to ensure that all nodes operate on the same
+database.
+
+Refer to the DuckDB documentation for tips on [securing DuckDB](
```

Comment:
Honestly I am not sure if you can secure DuckDB sufficiently. We might need to figure out more details for example around using query table function with functions that read from the file system .. is that possible? If yes.. should we add more info there?

Reply:
But for now .. I think we are ok to leave that problem to the user.